app/models/users_project.rb

```diff
@@ -18,4 +18,6 @@ class UsersProject < ApplicationRecord
   has_many :metrics, as: :ownable, dependent: :destroy
 
   validates :user_id, uniqueness: { scope: :project_id }
+
+  delegate :name, to: :user
```

Comment:
can we prefix it? I mean, `name` could also be delegated to `project` and it would make sense 😅